proxy/api/src/project.rs

```diff
@@ -161,6 +225,18 @@ impl Projects {
 
         Ok(projects)
     }
+
+    /// Helper method for logging the error as a warning and pushing the failure onto the failure
+    /// list.
+    fn handle_failure(
+        project: Project,
+        projects: &mut Self,
```

Comment:
Any reason why this isn't `handle_failure(&mut self, ...)` ie. a method on `Self`?

Reply:
Because I'm a silly billy